docs/configuration.md

```diff
@@ -62,6 +63,7 @@ User should chose following parameters combination (`clusterNetwork`+`defaultNet
 * `systemNamespaces` ([]string, optional): list of namespaces for Kubernetes system (namespaces listed here will not have `defaultNetworks` added)
 * `multusNamespace` (string, optional): namespace for `clusterNetwork`/`defaultNetworks`
 * `delegates` ([]map,required): number of delegate details in the Multus
+* `allowTryDeleteOnErr` (bool, optional): Enable or disable delegate DEL message to next when some missing error. Defaults to false.
```

Comment:
we'd like to consider a different name for this functionality, one idea is: `retryDeleteOnError` but maybe we should get some other opinions as well.

Reply:
good idea !